playwright/pages/shippingMethodsPage.ts

```diff
@@ -83,13 +86,35 @@ export class ShippingMethodsPage {
     });
   }
 
+  async gotoExistingShippingRate(shippingMethodId: string, shippingRateId: string) {
+    const existingShippingRateUrl = `${URL_LIST.shippingMethods}${shippingMethodId}/${shippingRateId}`;
+
+    await console.log(
+      `Navigates to existing shipping rate page: ${existingShippingRateUrl}`,
+    );
+
+    await this.page.goto(existingShippingRateUrl);
+
+    await this.shippingRateNameInput.waitFor({
+      state: "visible",
+      timeout: 10000,
+    });
+  }
+
+
   async clickCreateShippingZoneButton() {
     await this.createShippingZoneButton.click();
   }
 
-  async clickDeleteShippingMethod() {
-    await this.priceBasedRatesSection.locator(this.deleteShippingMethodButton).click();
+  async clickDeleteShippingRateFromTheList() {
+    await this.priceBasedRatesSection.locator(this.deleteShippingRateButtonOnList).click();
```

Comment:
Is this not gonna be confusing when deleting weight base shipping?

Reply:
Would clickDeletePriceBasedShippingMethod be more obvious and descriptive?

Reply:
https://github.com/saleor/saleor-dashboard/pull/4625/commits/3f2bca7a34276fee0d0f6a299d359f77124979cb